cmd/cluster-agent/subcommands/start/command.go

```diff
@@ -168,8 +169,14 @@ func start(log log.Component, config config.Component, telemetry telemetry.Compo
 		}
 	}
 
+	// Setup the leader forwarder for language detection and cluster checks
+	var leaderForwarder *apidca.LeaderForwarder
+	if pkgconfig.Datadog.GetBool("cluster_checks.enabled") || pkgconfig.Datadog.GetBool("language_detection.enabled") {
+		leaderForwarder = apidca.NewLeaderForwarder(pkgconfig.Datadog.GetInt("cluster_agent.cmd_port"), pkgconfig.Datadog.GetInt("cluster_agent.max_connections"))
```

Comment:
`cluster_agent.max_connections` will most likely be reached pretty fast and it might affect clc runners, should we have a separate leaderForwarder for both ? In that case we might have two connections between two DCA.